packages/ui/src/i18n/dictionaries/index.ts

```diff
@@ -5,3 +5,4 @@ export { frDict } from './fr';
 export { itDict } from './it';
 export { jaDict } from './ja';
 export { zhDict } from './zh';
+export { plDict } from './pl';
```

Comment:
exports seems to be ordered alphabetically. Can you put it between `jaDict` and `zhDict` instead, please?

Reply:
Done :-)